cpp/src/aztec/honk/sumcheck/sumcheck.test.cpp

```diff
@@ -70,7 +74,18 @@ TEST(Sumcheck, Prover)
         q_c, sigma_1, sigma_2, sigma_3, id_1,         id_2, id_3, lagrange_1
     };
 
-    auto transcript = Transcript(transcript::Manifest());
+    std::vector<transcript::Manifest::RoundManifest> manifest_rounds;
```

Comment:
Follow-on: Stuff like this and `mock_prover_contributions_to_transcript` could be replaced with the real manifest and the `mock_inputs_prior_to_challenge()` fctn

Reply:
Yeah, I think, more generally, we need to decide on a policy about when we use real things vs mocked things. @arijitdutta67 maybe you can keep this in mind as you work on the sumcheck tests and implement @ledwards2225's suggestion if you think it makes sense to do so?